### PR TITLE
VIP-911: copy updates for new report functionality

### PR DESCRIPTION
**Pull request by alice-dw**
new email:
```
The data you provided for ABCXYZ's election has been processed.

You can view an error report by logging into https://dashboard.votinginfoproject.org/ and clicking on the appropriate feed.

When looking at an error report, keep in mind that not all errors are cause for concern. The VIP team will let you know if there are errors that need addressing.

If we haven't already, our team will also let you know when the data is ready for your review in our staging/testing tool.

If you have any questions, please contact vip@democracy.works

Thank you!
```
https://democracyworks.slack.com/archives/CC4T6GBL4/p1678387140654539

### CODE REVIEW

File: notifications/sender.js [hunk 1]
```diff
@@ -186,7 +186,9 @@ module.exports = {
                       checksum +
                       "\nFull Error Report https://" +
                         process.env.BASE_URI + "/db/feeds/" + message.publicId + reportSuffix +
-                      "\nProcessing-Time (sec) " + duration);
+                      "\nProcessing-Time (sec) " + duration) + 
+                      "\nIf you encounter an error with the report link, make sure you are logged at https://" +
+                      process.env.BASE_URI;
```
[the remark this answers is not in the document]
Comment (danny-pease) [reply notifications/sender.js:191]: I like this!

**Review by danny-pease — APPROVED** [review]
📤

**Comment by alice-dw** [comment]
also noting i have done 0 testing on this. i have no idea how to trigger emails locally and figure its low stakes enough to just test in staging. if that is not a fair assumption then lmk and i can try to spend some time tinkering with it

**Review by dwxander — APPROVED** [review]
quotes look good to me!!!

**Pull request merged**